src/components/StatusBar.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FunctionComponent } from "react"
+import { View, StatusBar as RNStatusBar, StyleSheet } from "react-native"
+import { useSafeAreaInsets, EdgeInsets } from "react-native-safe-area-context"
+
+import { Colors } from "../styles"
+
+interface StatusBarProps {
+  backgroundColor?: string
+}
+
+const StatusBar: FunctionComponent<StatusBarProps> = ({
+  backgroundColor = Colors.primaryLightBackground,
+}) => {
+  const insets = useSafeAreaInsets()
```

Comment:
Really like moving this logic into this component! So much neater :)

Reply:
💯